pkg/typedef/bag.go

```diff
@@ -24,14 +24,14 @@ import (
 )
 
 type BagType struct {
-	Kind   string     `json:"kind"` // We need to differentiate between sets and lists
-	Type   SimpleType `json:"type"`
-	Frozen bool       `json:"frozen"`
+	ComplexType string     `json:"complex_type"` // We need to differentiate between sets and lists
+	Type        SimpleType `json:"type"`
```

Comment:
Since we are touching definition, could you please give this guy proper name
```suggestion
	ValueType        SimpleType `json:"value_type"`
```

Reply:
Done